lib/config/types.ts

```diff
@@ -411,6 +411,10 @@ export type MergeStrategy =
   | 'rebase-merge'
   | 'squash';
 
+// ref: https://github.com/renovatebot/renovate/issues/39458
+// This list should be added to as any new unsafe execution commands should be permitted
+export type AllowedUnsafeExecution = 'gradlewExecution';
```

Comment:
Given we don't yet want to (yet) implement this, maybe we can leave it as `undefined`?

```suggestion
export type AllowedUnsafeExecution = undefined;
```

Then when we add the first one, we can make sure it's then implemented? (by replacing `undefined` with a value)